src/plugins/upf/upf.h

```diff
@@ -490,6 +502,11 @@ typedef struct {
   vlib_main_t * vlib_main;
   vnet_main_t * vnet_main;
   ethernet_main_t * ethernet_main;
+
+  /* DPI apps hash */
+  uword* dpi_app_hash;
+  /* DPI apps vector */
+  upf_dpi_app_t *dpi_apps_pool;
```

Comment:
pls use ```upf_apps``` instead of dpi_apps_pool, the first use case will be dpi, but this will also be used for other stuff